code/modules/mob/living/carbon/alien/adult/queen.dm

```diff
@@ -181,3 +181,22 @@
 	if(!silent)
 		to_chat(user, span_noticealien("You discard [src]."))
 	return ..()
+
+/mob/living/carbon/alien/adult/royal/queen/attackby(obj/item/O, mob/user, params)
+	. = ..()
+	if(.)
+		return
+
+	if(istype(O, /obj/item/clothing/under/costume/maid) || istype(O, /obj/item/clothing/under/rank/civilian/janitor/maid))
+		if(do_after(user, 5 SECONDS, src))
+			maidify()
+			qdel(O)
+		return TRUE
+
+/mob/living/carbon/alien/adult/royal/queen/proc/maidify()
+	name = "alien queen maid"
+	desc = "Lusty, Sexy"
+	icon_state = "alienqmaid"
+	caste = "qmaid"
```

Comment:
it also looks like you took my code from that pr as is, that was my first ever pr and i didnt know how byond works so this var does absolutely nothing

Reply:
Yeah, 3am posting. <333

The name and sprite works, I need to get the desc to update.